CMakeLists.txt

```diff
@@ -1038,6 +1038,18 @@ IF(DOXYGEN_FOUND)
      ADD_CUSTOM_TARGET(apidoc ${DOXYGEN_EXECUTABLE} ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR} COMMENT "Generate the doxygen documentation into the doc directory.")
 ENDIF()
 
+IF((${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU" AND ${CMAKE_CXX_COMPILER_VERSION} VERSION_LESS 11.1) OR
+   (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang" AND ${CMAKE_CXX_COMPILER_VERSION} VERSION_LESS 14.0))
```

Comment:
Hmm... For Clang no need part `AND ${CMAKE_CXX_COMPILER_VERSION} VERSION_LESS 14.0` IMHO

Reply:
We do need it. Clang does implement these functions since v14, if we believe your reference. But AppleClang might not, or it has some other version numbering. This is what we need to find out.

Reply:
Please see logs for macOS and FreeBSD CI

Reply:
> We do need it. Clang does implement these functions since v14, if we believe your reference. But AppleClang might not, or it has some other version numbering. This is what we need to find out.

I see notification for `to_chars` method only